optaplanner-docs/src/main/asciidoc/Chapter-Planner_introduction/Chapter-Planner_introduction.adoc

```diff
@@ -149,7 +223,8 @@ $ cd examples
 $ runExamples.bat
 ----
 
-image::Chapter-Planner_introduction/distributionZip.png[align="center"]
+//image::Chapter-Planner_introduction/distributionZip.png[align="center"]
+// This image is redundant in the documentation. The steps to download, unzip and open are simple and clearly explained.
```

Comment:
Again I 'd like to keep the image to illustrate the directory tree of the expanded zip. I get your point - and it is a maintenance burden, but I find that people read the documentation diagonally and such illustrations do help to focus their attention as well as explain it better than some raw text.